.renovaterc.json

```diff
@@ -8,11 +8,31 @@
       "allowedVersions": "<10"
     },
     {
-      "description": "MIRSG Ansible roles",
-      "matchPackagePatterns": ["^mirsg"],
-      "versioning": "loose"
+      "matchPackageNames": [
+        "xnatdev/xnat-web",
+        "xnatdev/xsync",
+        "xnatx/ldap-auth-plugin",
+        "xnatdev/container-service",
+        "xnatx/xnatx-batch-launch-plugin",
+        "VUIIS/dax",
+        "icrimaginginformatics/ohif-viewer-xnat-plugin",
+        "xnatx/ml-plugin",
+        "xnatx/datasets-plugin",
+        "xnatdev/xnat-image-viewer-plugin",
+        "xnatx/xnatx-dxm-settings-plugin",
+        "NrgXnat/xnat-pipeline-engine"
+      ],
```

Comment:
I _think_ this should work as the `packageName` is taken from the `depNameTemplate` specified in the custom managers